auth_matrix/admin.py

```diff
@@ -40,13 +45,21 @@ def dehydrate(self, user):
         return data
 
     def export(self, queryset, *args, **kwargs):
```

Comment:
I looked at the docs and `queryset` can be None`, which is a bit odd, but either way it doesn't seem right to override `export` itself when there are `before_export` and `after_export` hooks provided by the library: https://django-import-export.readthedocs.io/en/latest/api_resources.html#import_export.resources.Resource.after_export

Instead, you should be using the `after_export` hook here, you receive the (filtered) queryset and tablib dataset as arguments so you have all the information you need.

You can override `get_queryset` to add the `group_names` annotation